lms/templates/navigation.html

```diff
@@ -151,7 +151,7 @@ <h1 class="logo">
   </nav>
 </header>
 % if course:
-<!--[if lte IE 8]>
+<!--[if lte IE 9]>
 <div class="ie-banner" aria-hidden="true">${_('<strong>Warning:</strong> Your browser is not fully supported. We strongly recommend using {chrome_link} or {ff_link}.').format(chrome_link='<a href="https://www.google.com/intl/en/chrome/browser/" target="_blank">Chrome</a>', ff_link='<a href="http://www.mozilla.org/en-US/firefox/new/" target="_blank">Firefox</a>')}</div>
```

Comment:
Since you're here, can you change
`http://www.mozilla.org/en-US/firefox/new/`
to `http://www.mozilla.org/firefox`? This will redirect people to their own language version of Firefox.

https://www.google.com/chrome is probably the better link for Chrome, although that still redirects to English, at least "en_US" isn't in the URL so if they start adding lang redirects we'll be ready.

Can you also do this in lms/templates/navigation-edx.html